website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Custom messages
+description: |-
+  Custom messages allow operators to configure system-wide messages.
+---
+
+# Custom Messages
+
+-> You must have **Vault 1.16.0 or higher** installed.
+
+Custom messages allow operators to configure system-wide messages and for vault users to view these messages.
+
+# Configure a Custom Message
+
+-> You must be a Vault operator or have capabilities to create a message.
```

Comment:
I'm not sure if we use the terminology "Vault operator" or admin to refer to folks who manage Vault.